### ISSUE TITLE
[Fretta] Derma_Hook ScorePanelHeaderLabel->DListView_DraggerBar cannot be Overriden over Scheme/Paint Hook

### ISSUE THREAD

**Issue by Vinzuerio** [Vinzuerio/ph-enhanced#52]
Resulting [LABEL] (Dbutton) seems stacked over DLabel.
This is due of December 2017 VGUI Derma Skins changes.

**Comment by daunknownman2010** [comment]
If anyone has potential workarounds for this issue, contributions would be wonderful! At the moment I've got no time to fully work on Garry's Mod projects due to real-life courses and Wolvin is unable to work properly because of outside factor issues.

**Comment by dhantasmic** [comment]
@daunknownman2010 @Vinzuerio Okay I spent a few hours on this but seem to have it figured out.  Would like someone else to test and confirm it works for them too.  I noticed when I did PrintTable(pnlCol) in the function below there are two sets of DButtons and DListView_DraggerBar.  I just disabled the first set, and everything seems to be back to normal with that.  Probably not the best fix, but good enough.  Hopefully this can help you guys out.


This is from fretta/gamemode/vgui/vgui_scoreboard_team.lua:
```
function PANEL:AddColumn( col )

	table.insert( self.Columns, col )
	
	local pnlCol = self.List:AddColumn( col.Name )
	
	if (col.iFixedSize) then pnlCol:SetMinWidth( col.iFixedSize ) pnlCol:SetMaxWidth( col.iFixedSize ) end
	if (col.HeaderAlign) then 
		pnlCol.Header:SetContentAlignment( col.HeaderAlign ) 
	end

	pnlCol:GetChildren()[1]:SetVisible(false)
	pnlCol:GetChildren()[2]:SetVisible(false)

	Derma_Hook( pnlCol, 	"Paint", 				"Paint", 	"ScorePanelHeader" )
	
	pnlCol.cTeamColor = team.GetColor( self.iTeam )
	
	Derma_Hook( pnlCol.Header, 	"Paint", 				"Paint", 	"ScorePanelHeaderLabel" )
	Derma_Hook( pnlCol.Header, 	"ApplySchemeSettings", 	"Scheme", 	"ScorePanelHeaderLabel" )
	Derma_Hook( pnlCol.Header, 	"PerformLayout", 		"Layout", 	"ScorePanelHeaderLabel" )
	
	pnlCol.Header:ApplySchemeSettings()

end
```

(I do have my scoreboard customized a bit in this picture)
![image](https://user-images.githubusercontent.com/2136915/36404392-5445f324-15b8-11e8-942f-2efbf6ecc4a8.png)

**Comment by daunknownman2010** [comment]
@dhantasmic Thanks for spending some time on looking into the issue! I'll have free time this weekend so I should be able to test it then hopefully. I'll respond back on the results.

**Comment by daunknownman2010** [comment]
@dhantasmic Your fix works on my end and has been added in 14j-staging branch with an appropriate comment that credits you for the fix. Thanks for the contribution and we look forward to potential future contributions if necessary.

**Comment by MichaelTait** [comment]
I know this is closed but I seem to still be seeing this issue. I have the updated version and correct code but overlap is present on the scoreboard headers. Kills, Ping etc all misplaced with 'label' wrote underneath. My coding knowledge is not on the stronger side though so haven't been able to solve this myself.
![capture](https://user-images.githubusercontent.com/23058227/38123498-3c5a69f0-33d3-11e8-9a26-7bf01c71cb6d.PNG)

**Comment by daunknownman2010** [comment]
@MichaelTait Are you sure you're using the absolute latest version (staging branch)? This fix was only added to the staging branch.

**Comment by daunknownman2010** [comment]
@MichaelTait I'm going to assume something's conflicting with the scoreboard as from your screenshot, it looks like something is applying the old scoreboard over the newer one. Have you tried looking at any addons on your server and disabling each one to see if it changes? Different Fretta gamemodes can also cause issues as well.

**Comment by MichaelTait** [comment]
@daunknownman2010 So I've tried a lot of stuff for a while now, still can't seem to fix the issue. The only prophunt gamemode I'm subscribed to is yours. I'm now wondering if it was a conflict through adding the gamemode as both an SVN and workshop file to the server. The gamemode seems to work by starting with default gamemode as prop_hunt when I have the gamemode in the collection the server uses. However, if I just add the gamemode using SVN so I can see the gamemode files in FTP then the gamemode refuses to start. Could it be that the workshop version is out of date and the code fixing this is missing? I did manage to get some files out from the SVN or workshop I think but the changes I made had no effect such as to autotaunt and implementing the above fix as it was missing. I'm using NFO Servers
